released/packages/coq-high-school-geometry/coq-high-school-geometry.8.11.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Laurent.Thery@inria.fr"
+homepage: "https://github.com/coq-community/HighSchoolGeometry/"
+license: "LGPL 2.1"
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/HighSchoolGeometry"]
+depends: [
+  "ocaml"
+  "coq" {>= "8.11" & < "8.13~"}
+]
+tags: [ "keyword: geometry" "keyword: teaching" "keyword: high-school" "category: Mathematics/Geometry/General" "date: 2004-01" ]
+authors: [ "Frédérique Guilhot <Frederique.Guilhot@sophia.inria.fr> [http://www-sop.inria.fr/lemme/Frederique.Guilhot/index.html]" ]
+bug-reports: "https://github.com/coq-community/HighSchoolGeometry/"
+dev-repo: "git+https://github.com/coq-community/HighSchoolGeometry/"
+synopsis: "Geometry for French high-school"
+description: """
+This library is dedicated to high-shool geometry teaching.
+The axiomatisation for affine euclidean space is in a non analytic setting."""
+flags: light-uninstall
```

Comment:
Here is the one generated via templates, which avoids some legacy declarations and has more precise license:
```suggestion
opam-version: "2.0"
maintainer: "Laurent.Thery@inria.fr"

homepage: "https://github.com/coq-community/HighSchoolGeometry"
dev-repo: "git+https://github.com/coq-community/HighSchoolGeometry.git"
bug-reports: "https://github.com/coq-community/HighSchoolGeometry/issues"
license: "LGPL-2.1-or-later"

synopsis: "Geometry in Coq for French high-school"
description: """
This Coq library is dedicated to high-shool geometry teaching.
The axiomatisation for affine Euclidean space is in a non analytic setting."""

build: [make "-j%{jobs}%" ]
install: [make "install"]
depends: [
  "coq" {>= "8.11" & < "8.13~"}
]

tags: [
  "category:Mathematics/Geometry/General"
  "keyword:geometry"
  "keyword:teaching"
  "keyword:high school"
  "logpath:HighSchoolGeometry"
]
authors: [
  "Frédérique Guilhot"
]
```